site2/docs/deploy-bare-metal.md

```diff
@@ -184,28 +184,28 @@ server.2=zk2.us-west.example.com:2888:3888
 server.3=zk3.us-west.example.com:2888:3888
 ```
 
-> If you have only one machine to deploy Pulsar, you just need to add one server entry in the configuration file.
+> If you only have one machine on which to deploy Pulsar, you only need to add one server entry in the configuration file.
 
-On each host, you need to specify the ID of the node in the `myid` file of each node, which is in each `data/zookeeper` folder of server by default (you can change the file location via the [`dataDir`](reference-configuration.md#zookeeper-dataDir) parameter).
+On each host, you need to specify the ID of the node in the `myid` file, which is in the `data/zookeeper` folder of each server by default (you can change the file location via the [`dataDir`](reference-configuration.md#zookeeper-dataDir) parameter).
 
 > See the [Multi-server setup guide](https://zookeeper.apache.org/doc/r3.4.10/zookeeperAdmin.html#sc_zkMulitServerSetup) in the ZooKeeper documentation for detailed information on `myid` and more.
 
-On a ZooKeeper server at `zk1.us-west.example.com`, for example, you can set the `myid` value like this:
+For example, on a ZooKeeper server like `zk1.us-west.example.com`, you can set the `myid` value as follows:
 
 ```bash
 $ mkdir -p data/zookeeper
 $ echo 1 > data/zookeeper/myid
 ```
 
-On `zk2.us-west.example.com` the command is `echo 2 > data/zookeeper/myid` and so on.
+On `zk2.us-west.example.com`, the command is `echo 2 > data/zookeeper/myid` and so on.
 
 Once you add each server to the `zookeeper.conf` configuration and have the appropriate `myid` entry, you can start ZooKeeper on all hosts (in the background, using nohup) with the [`pulsar-daemon`](reference-cli-tools.md#pulsar-daemon) CLI tool:
 
 ```bash
 $ bin/pulsar-daemon start zookeeper
 ```
 
-> If you plan to deploy zookeeper with bookie on the same node, you
+> If you plan to deploy Zookeeper with the Bookie on the same node, you
```

Comment:
ZooKeeper

Reply:
Fair enough. However, I noticed bookie was referenced in lower case in several locations and decided to match both the capitalization of Zookeeper with Bookie. What is your current differentiation rationale?